src/Game.tsx

```diff
@@ -99,6 +99,14 @@ function GameViewer({ games, showImport }: GameViewerProps): JSX.Element {
   }, []);
   const game = chosenGame || placeholderGame;
   const maxStep = game.board.timeline.length - 1;
+
+  // Upon importing for the first time, load the first game
+  useEffect(() => {
```

Comment:
I'm not keen on using an effect to trigger stuff on prop changes, i'm sure there's a better way to do this.

This approach also won't be updating the `Form.Select` to stay in sync.

It also occurs to me that loading in a new log file doesn't clear the old game state.

This is suggesting to me that the states are on the wrong components, and need to move up a bit, probably onto `App`.

This might also help with some upcoming refactoring of this file to separate the `GameViewer` from the `GameSelector`, `GameScroller` and `Scoreboard` components

Reply:
On second thought, this change is self-contained enough that even if it doesn't fit the model, it doesn't harm anything.

If you remove the `yarn.lock` i'll get this merged as-is.

Reply:
Yeah, it probably makes more sense to pull the `games` state up to `app` instead of using an effect like this, but I didn't want to futz with the state structure too much for my first PR :P